doc/code/qml_drawer.rst

```diff
@@ -53,6 +53,9 @@ Currently Available Styles
 .. |sks| image:: ../_static/style/sketch_style.png
     :width: 100%
 
+.. |sks| image:: ../_static/style/pennylane_style.png
```

Comment:
@ikurecic has this file been added? I can't see it in the commits 🤔

Reply:
You're 100% right, I forgot.